bjoern/25compat.h

```diff
@@ -2,4 +2,7 @@
 #define NOP do{}while(0)
 #define PyFile_IncUseCount(file) NOP
 #define PyFile_DecUseCount(file) NOP
+#if PY_MAJOR_VERSION >= 3
```

Comment:
I think we can drop everything below 2.7, but maybe that's better suited for another PR, not sure.